doc/api/util.md

```diff
@@ -348,8 +348,9 @@ changes:
     line. Defaults to 60 for legacy compatibility.
 
 The `util.inspect()` method returns a string representation of `object` that is
-primarily useful for debugging. Additional `options` may be passed that alter
-certain aspects of the formatted string.
+intended for debugging. The output of `util.inspect` may change at any time
+and should not be depended upon programmatically. Additional `options` may be
+passed that alter certain aspects of the formatted string.
```

Comment:
Somehow this sounds a bit harsh to me. Would you be fine to change it to something like

```md
The `util.inspect()` method returns a string representation of `object` that is
primarily intended for debugging. Additional `options` may be passed that alter
certain aspects of the formatted string.
Note that the output of `util.inspect` may undergo minor changes from time to
time without a semver-major step. Programmatically relying on the output is
therefore not recommended.
```

Reply:
i kinda want it to be harsh, there's a reason stuff like `internalBinding` has to be a thing now

Reply:
With your comment in place as is I would argue that landing #17576 as new default without putting it behind a option could be done as a semver-patch. And I do not see that at all.

Reply:
>With your comment in place as is I would argue that landing #17576 as new default without putting it behind a option could be done as a semver-patch. And I do not see that at all.

I'd rather we be harsher in the docs , but more conservative when actually landing possibly breaking PRs, so +1 on keeping this wording, but making #17576 `major`.

Realistically we know that if enough people depend on the output of `util.inspect`, no matter how internal, we won't be able to change it. But if this warning helps dissuade people from relying on that output, then that seems like a good thing.

Also if this PR makes #17576 a patch, that would make this PR semver-major 😁 .